lib/net.c

```diff
@@ -78,6 +71,51 @@ static struct rte_eth_conf gatekeeper_port_conf = {
 	},
 };
 
+/*
+ * @src_port and @dst_port must be in big endian.
+ * By specifying the tuple (proto, src_port, dst_port),
+ * it can filter both IPv4 and IPv6 addresses.
+ */
+int
+ntuple_filter_add(uint8_t portid,
+	uint16_t src_port, uint16_t dst_port, uint16_t queue_id)
+{
+	int ret = 0;
+	struct rte_eth_ntuple_filter filter = {
```

Comment:
Sure. I will add one filter for IPv4 with destination IP address and UDP ports, and another filter filter for IPv6 with only UDP ports.